webpack/assets/javascripts/react_app/components/ReportTemplate/Inputs/ReportAutoComplete.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FieldLevelHelp } from 'patternfly-react';
+
+import Form from '../../common/forms/CommonForm';
+import AutoComplete from '../../AutoComplete';
+
+const ReportAutoComplete = ({
+  label,
+  resourceType,
+  id,
+  initialQuery,
+  initialError,
+  info,
+  isRequired,
+  url,
+}) => (
+  <Form
+    label={label}
+    error={initialError}
+    required={isRequired}
+    tooltipHelp={info && <FieldLevelHelp content={<div>{info}</div>} />}
+  >
+    <AutoComplete
+      id={`report-input-${id}`}
+      inputProps={{
+        name: `report_template_report[input_values][${id}][value]`,
+      }}
+      useKeyShortcuts={false}
+      controller={resourceType}
+      url={url}
+      initialQuery={initialQuery}
+      initialError={initialError}
```

Comment:
I know that you took this example from the FormAutocomplete PR, though initialError is not yet supported by the current Autocomplete, very soon I will add this functionality.